### DISCUSSION TITLE
Changing SCSS module case builds locally, fails deployment to Vercel

### DISCUSSION THREAD

**Discussion by bscaspar** [vercel/next.js#25299] [Help] [answered]
### What version of Next.js are you using?

10.2.1

### What version of Node.js are you using?

10.23.0

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

Github -> Vercel

### Describe the Bug

1. create a `create-next-app` application, added 'sass', updated the `home.module.css` stylesheet to `home.module.scss` and deployed with `git push` to the `dev` branch which is connected to Vercel.
2. Change the name of `home.module.scss` to `Home.module.scss` and update the corresponding import in `pages/index.js` to `import styles from '../styles/Home.module.scss';`
3. `yarn build` runs with no errors
4. `git push` and deploy on Vercel
5. Build error in vercel:
```

13:44:00.195 | Failed to compile.
-- | --
13:44:00.196 | ModuleNotFoundError: Module not found: Error: Can't resolve '../styles/Home.module.scss' in '/vercel/path0/pages'
13:44:00.196 | &gt; Build error occurred
13:44:00.197 | Error: &gt; Build failed because of webpack errors
13:44:00.197 | at /vercel/path0/node_modules/next/dist/build/index.js:17:924
13:44:00.197 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:6:584)
13:44:00.217 | error Command failed with exit code 1.
13:44:00.217 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
13:44:00.234 | Error: Command "yarn run build" exited with 1

```

6. Redeploy from the web interface and leave "include build cache" unchecked results in the same error
7. Change the import in `pages/index.js` to `home.module.scss` but leave the actual file as `Home.module.scss` and it deploys fine

This feels like a really simple issue, am I missing something extremely basic?

### Expected Behavior

It builds as expected. Test repo here: [https://github.com/bscaspar/change-sass-case-test](https://github.com/bscaspar/change-sass-case-test)

### To Reproduce

Open up the test repo and follow the instructions. Test repo here: [https://github.com/bscaspar/change-sass-case-test](https://github.com/bscaspar/change-sass-case-test)

**Comment by timneutkens** [comment, marked as answer]
This is because your local filesystem is case-insensitive and your git is probably not configured to be case-sensitive. You'll likely want to run `git mv home.module.scss Home.module.scss` to make sure the casing update is saved in git.

**Reply by bscaspar** [reply]
ty!!
